Sources/Core/log.swift

```diff
@@ -5,10 +5,13 @@
 // See the LICENSE file for more information.
 
 import Foundation
-import os
+import os.log
 
 func log(_ type: LogType, _ message: StaticString, _ args: CVarArg...) {
-    os_log(type.osLogType, message, args)
+    if #available(iOS 14.0, *) {
+        let logger = Logger(subsystem: Bundle.frameworkIdentifier + ".network", category: "Log")
```

Comment:
That's not a network subsystem (it's a global function). If you decide to remove that function, that comment doesn't matter in another case we should remove it.

```suggestion
        let logger = Logger(subsystem: Bundle.frameworkIdentifier, category: "Log")
```